examples/client/pkg/lifecycle_v1alpha2/lifecycle_v1alpha2.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019 VMware, Inc. All Rights Reserved.
```

Comment:
So far, we have decided in the spec that the unique ID is the Name, not the FQDN. All references to FQDN as uniqueID must change to Name. 

From the spec: Adds a federated service to the consumer platform Then, the federated service can be discovered by the consumer platform as a local service. A FederatedService is uniquely identified in the owner platform by its name.

Reply:
In this MR we are not looking inside the content of federated_service type, so the processing of the uniqueness is left out of the scope. The resource_id field newly added will the primary id used for uniquness. 
Is that reasonable ?